lib/widgets/dashboard_widgets/transaction_info_card.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import '../read_times_tamp.dart';
+import '../transaction_avatar.dart';
+import 'package:seeds/utils/string_extension.dart';
+import 'package:seeds/widgets/read_times_tamp.dart';
+import 'package:seeds/widgets/transaction_avatar.dart';
+import 'package:seeds/i18n/wallet.i18n.dart';
+
+class TransactionInfoCard extends StatelessWidget {
+  final String transactionProfileAccount;
+  final String transactionProfileNickname;
+  final String transactionProfileImage;
+  final String transactionTimestamp;
+  final String transactionAmount;
+  final String transactionTypeIcon;
+  final GestureTapCallback callback;
+
+  const TransactionInfoCard({
+    Key key,
+    @required this.transactionAmount,
+    @required this.callback,
+    @required this.transactionProfileAccount,
+    @required this.transactionProfileNickname,
+    @required this.transactionProfileImage,
+    @required this.transactionTimestamp,
+    @required this.transactionTypeIcon,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: callback,
+      child: Container(
+        padding: EdgeInsets.only(top: 10, bottom: 10, right: 20, left: 20),
+        child: Row(
+          children: <Widget>[
+            TransactionAvatar(
```

Comment:
Is this avatar the same one we use in Profile? 
If its not, Can they be? 

If they can be, Can we make it generic and reuse in both places instead of creating multiple ones? 

@RaulUrtecho  @gguij004

Reply:
I want to make sure fo two things: 

1 -  We do not create custom widgets when its not absolutely needed.
2- We reuse widgets as much as possible.

@gguij004 @RaulUrtecho

Reply:
The truth is, I am very much at creating my customs widgets,I thing it depends on the functionality you are looking for in the ProfileAvatar, In my case it uses cache network image and if this does not exist it shows a color with the initials of the nickname

Reply:
whichever we use, The one you created or the one is already created, We need to pick one and reuse that one. Instead of creating a new one for every screen.

Reply:
Ok I can take a look at it :)